### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jrd656** [WebAhead5/SmartCooking#10]
Accessibility Checklist:

_I've commented against the checklist in italics. The non-italic text is just taken from a checklist I found online_

1. Screen-reader Compatible Web Pages
Web pages must be readable by electronic “screen readers” — which are devices commonly used by the visually impaired to surf the internet.

_Screen reader works for the inputs. I'm not sure it should be able to read the images and forms?_

2. Alt-tagged Images
Images must have “alt tags” (a type of metadata) which will help visually-impaired users to identify the image via text or via a screen reader. Search engine optimizers will know of alt-tags as a way to optimize images for ranking high in ‘Google Images’.

_images don't currently have alt tags. Suggest these are added but I guess it will be difficult?_

3. Alt-tagged Tables
If you’re using tables on any web pages, they must also have alt-tags — similar to those used with images — to help explain each column via text. The user’s screen-reader will then read that text aloud, describing the contents of the table.   

_No tables. Not necessary to change_

4. Automatic Scripting
Any scripted display usage of image mapping should be accompanied by textual alternatives.

_not relevant_

5. Style Sheet Independence
Your web pages shouldn’t need a stylesheet to display in a way that is understandable. They should be readable to screen readers, most of which cannot load style sheets.

_seems readable enough. The image goes massive, but I don't know what you're supposed to do about that?_

6. Accessible Forms
Web forms should be usable even without a mouse, and each field should be labeled.

_ran out of time to check the rest. Please see below! _

7. Text Links to Plugins
If your website requires a specific plugin to work correctly, you should present the link to that plugin in text form.  

8. Inclusive Color Schemes
Your website should not contain any colors or color combinations that are either integral to the correct usage of your website, or which may confuse those with color blindness in any way pertaining to your website, products or services.

9. Keyboard-friendly Browsing
Your website shouldn’t be reliant on the usage of a mouse or touchpad. Users who wish to navigate using only keyboard keys should be able to do so.

10. Harmless Website Design
Your website should not serve images or videos that can cause seizures.

**Comment by jrd656** [comment]
Oh, if you go to the audits tab in developer tools, you can see a good run down:
![image](https://user-images.githubusercontent.com/51966598/77629661-9077e780-6f52-11ea-9ecc-57375dfff8cf.png)

Click accessibility for a breakdown
